modules/server/src/main/java/org/jboss/wsf/stack/cxf/security/authentication/SubjectCreatingPolicyInterceptor.java

```diff
@@ -163,7 +149,7 @@ protected Principal getPrincipal(Principal originalPrincipal, Subject subject)
       if (!principals.isEmpty())
       {
           Principal principal = principals.iterator().next();
-          if (groupClass == null || !groupClass.isInstance(principal))
+          if (!(principal instanceof Group))
```

Comment:
Same for https://issues.redhat.com/browse/JBWS-4133.

Reply:
changed